parlai/mturk/core/dev/agents.py

```diff
@@ -298,52 +243,21 @@ def get_connection_id(self):
         """Returns an appropriate connection_id for this agent"""
         return "{}_{}".format(self.worker_id, self.assignment_id)
 
-    def log_reconnect(self):
```

Comment:
Reconnects and related state tracking are no longer needed as they are handled on the server-side